src/main/java/rx/Observer.java

```diff
@@ -34,7 +34,7 @@
      * <p>
      * The {@link Observable} will not call this method if it calls {@link #onError}.
      */
-    public abstract void onCompleted();
+    void onCompleted();
```

Comment:
Both modifiers are implicit and redundant. Run `javap` on the class file and you'll see both are still present.